lib/spreedly.ex

```diff
@@ -230,6 +230,21 @@ defmodule Spreedly do
     get_request(env, list_transactions_path(), params)
   end
 
+  @doc """
+  Retrieve a list of all gateways in the environment with optional
```

Comment:
Let's document that this is listing `created` gateways. `All gateways` to me implies that it might be the list of gateway options.

Reply:
Alright. I didn't realize that it could imply gateway options. I'll make the change.